modules/serverpod_auth/serverpod_auth_server/test/firebase/firebase_auth_test.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:serverpod_auth_server/src/firebase/exceptions/firebase_exception.dart';
+import 'package:serverpod_auth_server/src/firebase/firebase_auth_manager.dart';
+import 'package:test/test.dart';
+
+import 'firebase_auth_mock.dart';
+
+void main() {
+  group(
+    'Given a Firebase Auth class with a valid UserRecord, ',
+    () {
+      late FirebaseAuthManager auth;
+
+      setUp(() async {
+        auth = FirebaseAuthManager(
+          testAccountServiceJson,
+          authClient: MockAuthBaseClient(
+            userJson: getUserRecord(
+              uuid: 'abcdefghijklmnopqrstuvwxyz',
+              validSince: DateTime.now().subtract(const Duration(days: 1)),
+            ),
+          ),
+          openIdClient: MockTokenClient(),
+        );
+      });
+
+      test(
+        'when calling verifyIdToken with a valid idToken, then the returned idToken should match',
```

Comment:
Suggestion/Discussion(Nitty): I usually avoid using phrases such as `should` since we know exactly what we want. Use a more direct phrasing around what is returned. `then the returned idToken matches the valid idToken.`.

This applies to all tests in this file.